src/test/java/org/sqlite/DBMetaDataTest.java

```diff
@@ -1669,4 +1667,593 @@ public void getPrimaryKeys(String table) throws SQLException {
+        assertThat(rsmeta.getColumnCount()).isEqualTo(2);
+        assertThat(rsmeta.getColumnName(1)).isEqualTo("TABLE_SCHEM");
+        assertThat(rsmeta.getColumnName(2)).isEqualTo("TABLE_CATALOG");
+
+        rs = meta.getIndexInfo(null, "nonexistent", null, false, false);
+        assertThat(rs).isNotNull();
+        assertThat(rs.next()).isFalse();
+
+        rs = meta.getTables(null, "nonexistent", null, null);
+        assertThat(rs).isNotNull();
+        assertThat(rs.next()).isFalse();
+        rsmeta = rs.getMetaData();
+        assertThat(rsmeta.getColumnCount()).isEqualTo(10);
+        assertThat(rsmeta.getColumnName(1)).isEqualTo("TABLE_CAT");
+        assertThat(rsmeta.getColumnName(2)).isEqualTo("TABLE_SCHEM");
+        assertThat(rsmeta.getColumnName(3)).isEqualTo("TABLE_NAME");
+        assertThat(rsmeta.getColumnName(4)).isEqualTo("TABLE_TYPE");
+        assertThat(rsmeta.getColumnName(5)).isEqualTo("REMARKS");
+        assertThat(rsmeta.getColumnName(6)).isEqualTo("TYPE_CAT");
+        assertThat(rsmeta.getColumnName(7)).isEqualTo("TYPE_SCHEM");
+        assertThat(rsmeta.getColumnName(8)).isEqualTo("TYPE_NAME");
+        assertThat(rsmeta.getColumnName(9)).isEqualTo("SELF_REFERENCING_COL_NAME");
+        assertThat(rsmeta.getColumnName(10)).isEqualTo("REF_GENERATION");
+
+        rs = meta.getColumns(null, "nonexistent", null, null);
+        assertThat(rs).isNotNull();
+        assertThat(rs.next()).isFalse();
+        rsmeta = rs.getMetaData();
+        assertThat(rsmeta.getColumnCount()).isEqualTo(24);
+        assertThat(rsmeta.getColumnName(1)).isEqualTo("TABLE_CAT");
+        assertThat(rsmeta.getColumnName(2)).isEqualTo("TABLE_SCHEM");
+        assertThat(rsmeta.getColumnName(3)).isEqualTo("TABLE_NAME");
+        assertThat(rsmeta.getColumnName(4)).isEqualTo("COLUMN_NAME");
+        assertThat(rsmeta.getColumnName(5)).isEqualTo("DATA_TYPE");
+        assertThat(rsmeta.getColumnName(6)).isEqualTo("TYPE_NAME");
+        assertThat(rsmeta.getColumnName(7)).isEqualTo("COLUMN_SIZE");
+        assertThat(rsmeta.getColumnName(8)).isEqualTo("BUFFER_LENGTH");
+        assertThat(rsmeta.getColumnName(9)).isEqualTo("DECIMAL_DIGITS");
+        assertThat(rsmeta.getColumnName(10)).isEqualTo("NUM_PREC_RADIX");
+        assertThat(rsmeta.getColumnName(11)).isEqualTo("NULLABLE");
+        assertThat(rsmeta.getColumnName(12)).isEqualTo("REMARKS");
+        assertThat(rsmeta.getColumnName(13)).isEqualTo("COLUMN_DEF");
+        assertThat(rsmeta.getColumnName(14)).isEqualTo("SQL_DATA_TYPE");
+        assertThat(rsmeta.getColumnName(15)).isEqualTo("SQL_DATETIME_SUB");
+        assertThat(rsmeta.getColumnName(16)).isEqualTo("CHAR_OCTET_LENGTH");
+        assertThat(rsmeta.getColumnName(17)).isEqualTo("ORDINAL_POSITION");
+        assertThat(rsmeta.getColumnName(18)).isEqualTo("IS_NULLABLE");
+        // should be SCOPE_CATALOG, but misspelt in the standard
+        assertThat(rsmeta.getColumnName(19)).isEqualTo("SCOPE_CATLOG");
```

Comment:
this was fixed in 4429515442f999457e0517216c1e5e0c5b14cb09 and thus should be changed